.changeset/strange-kids-change.md

```diff
@@ -0,0 +1,10 @@
+---
+'@emotion/core': minor
+'@emotion/styled': minor
+'@emotion/styled-base': minor
```

Comment:
Why `minor` instead of `patch`? Don't mind strongly either way, just curious

Reply:
I've decided to do the same for the types-related change. Kinda doesn't matter but maybe puts less pressure on people to immediately update their code. At the time being they **don't** have to do the update - they will only have to do that if they start using React 18 and that has not been even released yet. 

Dunno, I just sometimes use minor bumps for some additional context than a "new feature". The problem here is that this might affect some people - for example, the serialized output of `jest-emotion` might change (like it has changed in our tests on this branch here). So this leaves a little bit of breathing room for people so they can stay on the previous version if they want.

Reply:
Sounds good